test/files/run/t12799/C.java

```diff
@@ -0,0 +1,11 @@
+
+package example;
+
+@Deprecated(since="you like it")
```

Comment:
Didn't take arguments back on Java 8. We can make this test `// javaVersion: 9+`

Reply:
oh right. You always say that, yet I always forget. What is Java 8 again?

Reply:
For the record, made everything a single test because Swiss frugality, but I could also split out the last test. In fact, I will do that because jdk 8 is a major platform.